rd-net/RdFramework/Impl/RdSignal.cs

```diff
@@ -85,7 +85,7 @@ public void Fire(T value)
       wire.NotNull(this).Send(RdId, SendContext.Of(SerializationContext, value, this), (sendContext, stream) =>
       {
         var me = sendContext.This;
-        SendTrace?.Log($"{me} :: value = {sendContext.Event.PrintToString()}");
+        SendTrace?.Log($"{me} :: value = {sendContext.Event?.PrintToString()}");
```

Comment:
_(Same as above)_ `PrintToString` is supposed to handle null better (by printing `"<null>"`), so could we leave this as is?